### DISCUSSION TITLE
How to make a horizontal bar stack chart fully expanded

### DISCUSSION THREAD

**Discussion by jimmy-e** [airbnb/visx#1199] [Q&A]
This was somewhat commented in [this issue](https://github.com/airbnb/visx/issues/1053), but I'm still a bit confused; what exactly would I need to do to take [this horizontal bar stack chart](https://airbnb.io/visx/barstackhorizontal) and make it be fully expanded like seen in the XY Chart?

**Comment by williaster** [comment]
Hey @jimmy-e you're looking for the `expand` prop which you can find in the `@visx/shape` documentation [here](https://airbnb.io/visx/docs/shape#BarStackHorizontal_offset). You can read even more about how stacks work in the [`d3-shape` docs](https://github.com/d3/d3-shape#stacks).

Here's a [sandbox](https://codesandbox.io/s/divine-rgb-h4p22?file=/Example.tsx:3013-3045) (note in this example you also need to update the scale domain to be `0-1` rather than in temperature units)

**Reply by VigneshTrantor** [reply]
You can do 
`<BarStackHorizontal offset={"expand"} >{...children}</ BarStackHorizontal>` as suggested by williaster
